docs/devtools.md

```diff
@@ -75,3 +75,95 @@ Note
 This example is pretty-printed for readability. When setting the
 enviroment variable, it should be collapsed to one single line.
 :::
+
+## Trino Tips
+
+Trino is an SQL query engine, but not an SQL database. What does that mean? Traditional databases
```

Comment:
Do these paragraphs need to be hard wrapped? It's generally preferable to let the text editor do the wrapping.